AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.m

```diff
@@ -369,9 +384,22 @@ - (void)trackPage:(NSString *)pageName withProperties:(NSDictionary<NSString *,
 }
 
 - (void)sendLog:(id<MSLog>)log flags:(MSFlags)flags {
+  [(flags & MSFlagsCritical) != 0 ? self.criticalChannelUnit : self.channelUnit enqueueItem:log flags:flags];
```

Comment:
nitpick: for readability, please consider split into 2 lines instead of inlining.